backend/packages/Upgrade/src/env.ts

```diff
@@ -89,6 +89,7 @@ export const env = {
   initialization: {
     contextMetadata: JSON.parse(getOsEnv('CONTEXT_METADATA')),
     adminUsers: parseAdminUsers(getOsEnv('ADMIN_USERS')),
+    metrics: JSON.parse(getOsEnv('METRICS')),
```

Comment:
`getOsEnv` means that it should not be null. This will throw error if in environment variable is not defined. And also what if the format is incorrect `JSON.parse` will throw error. Please take care of those scenarios

Reply:
okay.

Reply:
done